src/components/CivicLikerV3/Dashboard.vue

```diff
@@ -3,7 +3,7 @@
     <div
       v-if="!validatorAddress"
       key="loading"
-      class="flex items-center justify-center min-h-[180px]"
+      class="flex items-center justify-center min-h-[720px]"
```

Comment:
180px is for the height of the banner
<img width="580" alt="image" src="https://user-images.githubusercontent.com/17264716/207918730-570f20ba-d9e2-45ac-9384-d205b80aa88c.png">

So when the loading is completed, the spinner will looks like turning into the avatar
https://user-images.githubusercontent.com/17264716/207918815-1fa8a441-1b4c-4a38-b81a-6f5db975eca1.mov

I suggest wrapping the `<Transition/>` with `<div class="min-h-[720px]">` instead

Reply:
Also, I suggest providing `height:64px` for the Civic Liker logo here to prevent slightly shifting 
https://github.com/likecoin/liker-land/blob/a6749be921c56175359768f275b4641d8df02e20/src/components/CivicLikerV3/Page.vue#L9

Reply:
updated, please check if I understood correctly